packages/genesys-spark-components/src/components/beta/gux-form/example.html

```diff
@@ -0,0 +1,50 @@
+<gux-form-beta>
+  <form>
```

Comment:
This is more just a question for my own understanding. Why create a gux-form-beta that's just a slot rather than having it contain a form tag?

Reply:
If I was to to put the form tag inside of `gux-form-beta` developers would not be able to access the the form tag to perform actions they may need to do as it would be in the `shadowDOM`.